pkg/gomoney/currency.go

```diff
@@ -4,6 +4,15 @@ type Currency string
 
 const (
 	USD Currency = "USD"
-	RUB          = "RUB"
-	NGN          = "NGN"
+	RUB Currency = "RUB"
+	NGN Currency = "NGN"
 )
```

Comment:
Goland actually saw no problem with only providing type for the first enum, but VSCode was screaming at me. 
The code compiles when only one type is given anyways and i think this is how the go style guide recommends it, but okay. This works, for vscode's sake ))